colibris/conf/settings.py

```diff
@@ -31,7 +31,9 @@
 TASK_QUEUE = {}
 EMAIL = {}
 
-API_DOCS_PATH = '/api/docs'
+API_DOCS_URL = '/api/docs'
+APISPEC_URL = '{}/apispec'.format(API_DOCS_URL)
+DOCS_STATIC_URL = '{}/static'.format(API_DOCS_URL)
```

Comment:
@ccrisan, I don't  like this way, but I'm not sure if we have a better option if we go with this approach. We use these settings to inject them in swagger html template. Any other idea?